src/Spain/Mews.Fiscalizations.Spain/Model/Request/Invoice.cs

```diff
@@ -12,7 +12,7 @@ public class Invoice
             String0To500 description,
             TaxBreakdown taxBreakdown,
             bool issuedByThirdParty,
-            CounterPartyCompany counterPartyCompany)
+            CounterParty counterParty)
```

Comment:
Are we sure it doesn't need to be a company?
Also, how about we call it assignee?

Reply:
I have no idea why this was called `CounterPartyCompany`

We've been always providing either customer or company info (based on to who the invoice is assigned).

I thought of calling this Assignee but i was in doubt that other people might not really understand what it means :D